login.php

```diff
@@ -44,22 +44,22 @@
 					<h3><form method = "post" id= "01_login">
 						
 						<p style = "margin: 2.3vw 0vw 0vw 3.7vw; font-size: 1.4vw;">Username:</p>
-						<input style = "width: 18.8vw; height: 1.9vw; margin: 1vw 2vw 0vw 3.7vw; font-size: 1.2vw;" type = "text" name = "username" placeholder="Your Username..."/>
+						<input style = "width: 18.8vw; height: 1.9vw; margin: 1vw 2vw 0vw 3.7vw;" type = "text" name = "username" placeholder="Your Username..."/>

```

Comment:
I have removed the font-size from the three input fields - name = "username", "password", "submit" and incorporated it in the style sheet.